operations/http.go

```diff
@@ -82,11 +82,14 @@ func (ac *legacyClient) doReq(method, path string, apiVersion int, body io.Reade
 		return nil, err
 	}
 
-	req.Header.Add(evergreen.APIKeyHeader, ac.APIKey)
-	req.Header.Add(evergreen.APIUserHeader, ac.User)
 	if ac.JWT != "" {
 		req.Header.Add(evergreen.KanopyTokenHeader, "Bearer "+ac.JWT)
 	}
+
+	if ac.User != "" && ac.APIKey != "" {
```

Comment:
Just out of curosity, should this be coupled? i.e. will these two headers be completely removed in favor of the kanopy header?

Reply:
yes, they are not needed if we have a jwt token

Reply:
Ah got it, ty!